src/lazy/binary/raw/v1_1/struct.rs

```diff
@@ -229,21 +230,18 @@ impl<'top> RawBinaryStructIterator_1_1<'top> {
     /// If a value is parsed successfully, it is returned along with an [`ImmutableBuffer`]
     /// positioned after the value. If the value consists of NOPs, no value is returned but a
     /// buffer positioned after the NOPs is returned.
-    fn peek_value(
+    fn peek_value_expr(
```

Comment:
🗺️ Previously the lazy binary struct could only surface fields with a value literal. This method can now return either a `ValueLiteral` or an `EExp`.